pipelines/controller/README.md

```diff
@@ -1,2 +1,80 @@
-Please see
-[this Wiki page](https://github.com/google/fhir-data-pipes/wiki/Try-out-the-FHIR-Pipelines-Controller).
+# FHIR Pipelines Controller
+
+The FHIR Pipelines Controller helps you schedule and manage the transformation
+of data from a [HAPI FHIR server](https://hapifhir.io/) to a collection of
+Apache Parquet files. It uses the FHIR Data Pipes JDBC pipeline to run either
```

Comment:
Let's drop "JDBC" from here as it is going to be changed soon, i.e., both FHIR-search and JDBC will be supported.

Reply:
How soon? Ideally if this PR is merged now it reflects the current state, and this is updated when both are supported.